app/models/chef.rb

```diff
@@ -1,4 +1,5 @@
 class Chef < ApplicationRecord
    validates_presence_of :name
    has_many :dishes
+   has_many :ingredients
```

Comment:
The only way this association on number 4 will work is with a `through` association because there is no direct connection between a chef and an ingredient.

```ruby
class Chef < ApplicationRecord
  validates_presence_of :name
  has_many :dishes
  has_many :ingredients, through: :dishes
end
```